doc/content/gateways/dragino-lps8/_index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Dragino LPS8"
+description: ""
+weight: 
+---
+
+The **Dragino LPS8** is a LoRaWAN indoor gateway. Its operating system is based on the OpenWrt Linux system. The hardware includes an SX1308 LoRa concentrator that provides 10 programmable parallel demodulation paths. The gateway has an ethernet and WiFi backhaul. The Dragino LPS8 uses the Semtech legacy packet forwarder. The web-based management console allows you to configure and quickly connect the gateway with any LoRaWAN network. LPS8 supports EU868, IN865, US915, AU915, AS923, and KR920 ISM bands.
+
+This page guides you on how to connect the Dragino LPS8 to {{% tts %}}.
+
+![Dragino LPS8](dragino-lps8.png)
```

Comment:
```suggestion
{{< figure src="dragino-lps8.png" alt="Dragino LPS8" >}}
```

Use the image at https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.robotshop.com%2Fde%2Fde%2Fdragino-lps8-innen-lorawan-gateway-915-mhz.html&psig=AOvVaw0zPUdB0JeK548THnGi2VKu&ust=1604583632575000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCJia_fuB6ewCFQAAAAAdAAAAABAL